packages/dx-react-scheduler-material-ui/src/templates/day-view/row.test.jsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Row } from './row';
+
+describe('Week View Row', () => {
+  it('should pass rest props to the root element', () => {
+    const tree = shallow((
+      <Row data={{ a: 1 }}>
```

Comment:
What about the test that checks children rendering?

Reply:
done